2200022/14649.py

```diff
@@ -0,0 +1,14 @@
+# 14649 신용카드 만들기 1
+
+T = int(input())
+for i in range(T):
+    numbers = list(map(int, input().split()))
+    A = []
+    A = sum(numbers[0::2])*2
+    B = sum(numbers[1::2])
+    N = 0
+    if A + B + N % 10 == 0:
+        print(f"#{i+1} {N%10}")
+    elif A + B + N % 10 > 0: 
+        N = 10 - ((A + B) % 10)
+        print(f"#{i+1} {N%10}")
```

Comment:
잘 풀어주셨네요! 👍 
```suggestion
    N = 10 - ((A + B) % 10)
    print(f"#{i+1} {N%10}")
```
`+` 와 `%` 연산자 중 `%` 연산이 먼저 되기 때문에, 의도하신 것처럼 코드를 작성하려면 괄호를 활용해주셔야 해요!
또한, `N`을 구하기 위해 조건문을 나눠주지 않으셔도 정답을 구할 수 있습니다 😄